docs/src/markdown-pages/aptitudes/filesystem.md

```diff
@@ -1,17 +1,31 @@
 ---
-name: "Filesystem"
-links_js: "Filesystem"
-description: "Provides read and write access to the filesystem, enabling Loop Authors to read and write files."
+name: 'Filesystem'
+links_js: 'Filesystem'
+description: 'Provides read and write access to the filesystem, enabling Loop Authors to read and write files.'
 ---
+
 Provides read and write access to the filesystem, enabling Loop Authors to read and write files.
 
-[Permissions must be declared](https://github.com/open-olive/loop-development-kit/tree/main/ldk/javascript#loop-permissions) for a Loop to use the Filesystem Aptitude. 
+[Permissions must be declared](https://github.com/open-olive/loop-development-kit/tree/main/ldk/javascript#loop-permissions) for a Loop to use the Filesystem Aptitude.
 
 The filesystem permissions object must be populated with the locations you want to access in the following format:
+
 ```
 ldk:
  permissions:
    filesystem:
      pathGlobs:
        -  value: '/tmp/*.txt'
-```
\ No newline at end of file
+```
+
+### Filesystem Notes
+
+When your Loop is executing withing Olive Helps, it will generate its own unique "work directory" upon first run. This work directory is separate from every other Loop, and is empty at start.
+
+#### Other rules pertaining to Loop's unique "work directory":
+
+- Loops can do whatever they want in this directory, but they cannot destroy their "work directory" reference. For example, `filesystem.remove("./")` is **invalid**.
+- If your Loop accesses a relative path (example: `./some-file.txt`), you do not have to specify that path in your `package.json` permission list as a `pathGlob`.
+- All relative paths are automatically allowed (as long as they actually resolve to something in the Loop's "work directory").
```

Comment:
No need for parenthesis here: 
All relative paths are automatically allowed as long as they actually resolve to something in the Loop's "work directory".

Reply:
*high five*